docker-compose.yml

```diff
@@ -14,15 +14,15 @@ services:
     volumes:
       - rabbitmq:/var/lib/rabbitmq
     environment:
-      VIRTUAL_HOST: rabbitmq.it490.localhost
+      VIRTUAL_HOST: rabbitmq.local
```

Comment:
Could these be changed to `rabbitmq.*` so that any vhost, regardless of subdomain will be included?